pages/tag.vue

```diff
@@ -0,0 +1,58 @@
+<template>
+  <v-container>
+   <v-col>
+      <h2 class="mb-8">ADCIONAR PARTICIPANTE PELA TAG</h2>
+      {{ typeof idTag }}
+      <v-card class="left-border">
+        <v-card-text>
+          <v-text-field
+           label="TAG"
+           v-model.number="idTag"
+          />
+        </v-card-text>
+        <v-card-actions>
+          <v-btn @click="saveMember" color="primary">Adicionar</v-btn>
+        </v-card-actions>
+      </v-card>
+   </v-col>
+  </v-container>
+</template>
+
+<script>
+import gql from 'graphql-tag'
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/rafaelmbcosta/league-nuxt/pullRequest?prid=7513933)
```suggestion
import gql from "graphql-tag"
```